core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -20,15 +24,17 @@ object ControllerConfig {
       skewDetectionIntervalMs = Option(Constants.reshapeSkewDetectionIntervalInMs),
       statusUpdateIntervalMs =
         Option(AmberUtils.amberConfig.getLong("constants.status-update-interval")),
-      AmberUtils.amberConfig.getString("fault-tolerance.log-storage-type")
+      AmberUtils.amberConfig.getString("fault-tolerance.log-storage-type"),
+      None
```

Comment:
can you add the parameter keyword here for readability?

Reply:
done